app/controllers/verify/review_controller.rb

```diff
@@ -23,6 +23,14 @@ def confirm_current_password
     def new
       idv_session.params.symbolize_keys!
       analytics.track_event(Analytics::IDV_REVIEW_VISIT)
+
+      phone_of_record_msg = ActionController::Base.helpers.content_tag(
+        :strong, t('idv.messages.phone.phone_of_record')
+      )
+
+      # TODO: why doesn't this correctly output html without `html_safe`?
```

Comment:
I'm not sure how to best address this. Without calling `html_safe` the tags inside the translation helper get interpolated as a string, not html. The key in the locale file includes the `_html` postfix but it doesn't seem to be working.

Rubocop recommends safe_join, which obviously doesn't work since the content is a string.